Tools.mk

```diff
@@ -1,5 +1,5 @@
 bazelisk@v     := github.com/bazelbuild/bazelisk@v1.11.0
-buildifier@v   := github.com/bazelbuild/buildtools/buildifier@4.2.5
+buildifier@v   := github.com/bazelbuild/buildtools/buildifier@5.0.1
```

Comment:
This is updated to bust the cache since currently, we don't cache per version. But we need to get a "new" clang.